subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/configurationcache/ConfigurationCacheFixture.groovy

```diff
@@ -258,7 +258,8 @@ class ConfigurationCacheFixture {
         if (details.runsTasks) {
             spec.outputContains("Calculating task graph as no cached configuration is available for tasks:")
         } else {
-            spec.outputContains("Creating tooling model as no cached configuration is available for the requested model")
+            assert spec.getOutput().contains("Creating tooling model as no cached configuration is available for the requested model") ||
```

Comment:
👍 I think it might be useful to introduce a `contains(java.util.regex.Pattern)` but this is good enough

Reply:
We are also using a slightly different patter here:
https://github.com/gradle/gradle/blob/484ffb691aa4fb6e3790a823cae2ee0d688abef8/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/configurationcache/ConfigurationCacheFixture.groovy#L270-L297

It might be good to unify them. So, maybe something like `spec.outputContainsAny(list)`